src/app/onion/core/learning-object.service.ts

```diff
@@ -7,7 +7,7 @@ import { CookieService } from 'ngx-cookie';
 import { USER_ROUTES } from '@env/route';
 import { AuthService } from '../../core/auth.service';
 
-import { retry, catchError } from 'rxjs/operators';
+import { retry, catchError, map } from 'rxjs/operators';
```

Comment:
Why was map imported here? I don't see it used.

Reply:
Resolved in d4a5c36